frontend/app/src/components/defi/Borrowing.vue

```diff
@@ -44,15 +44,15 @@ const refresh = async () => {
   await defiLending.fetchBorrowing(true);
 };
 
-onMounted(async () => {
+onMounted(() => {
```

Comment:
Is there any reason we are removing the async/await here?

Reply:
it is causing i18n to break...